### PR TITLE
Improve error when SCC admission fails to create SecurityContextConstraintsProvider

### PR DESCRIPTION
**Pull request by sallyom**
@deads2k 
So the next person who messes with SCC admission has a better experience.

### CODE REVIEW

File: pkg/security/apiserver/admission/sccadmission/admission.go [hunk 1]
```diff
@@ -152,7 +152,7 @@ func (c *constraint) computeSecurityContext(a admission.Attributes, pod *coreapi
 	logProviders(pod, providers, errs)
 
 	if len(providers) == 0 {
```
Comment (deads2k) [inline pkg/security/apiserver/admission/sccadmission/admission.go]: This condition is different from sccLister.List() is empty. Please give me a different message above.

Comment (sallyom) [reply pkg/security/apiserver/admission/sccadmission/admission.go]: got it

File: pkg/security/apiserver/admission/sccadmission/admission.go [hunk 2]
```diff
@@ -133,6 +133,9 @@ func (c *constraint) computeSecurityContext(a admission.Attributes, pod *coreapi
 	if err != nil {
 		return nil, "", nil, admission.NewForbidden(a, err)
 	}
+	if len(constraints) == 0 {
+		return nil, "", nil, admission.NewForbidden(a, fmt.Errorf("no SecurityContextConstraints found in namespace %s", a.GetNamespace()))
```
Comment (deads2k) [inline pkg/security/apiserver/admission/sccadmission/admission.go]: wrong spot.  YOu can still have a message here about no matching constraints for namespace, but do a separate list above this.

Comment (sallyom) [reply pkg/security/apiserver/admission/sccadmission/admission.go]: only list all if didn't find any in the ns?

**Comment by sallyom** [comment]
ok, so now installs are failing, in events I see 21 times:
`"message": "Error creating: pods \"some-operator-7cd445d69-\" is forbidden: no SecurityContextConstraints found in cluster",`    
so.. that's not quite right.

**Pull request merged**